src/routes/delegates/delegates.service.ts

```diff
@@ -58,4 +63,25 @@ export class DelegatesService {
       results: delegates.results,
     };
   }
+
+  async postDelegates(
+    chainId: string,
+    createDelegateDto: CreateDelegateDto,
+  ): Promise<NetworkResponse> {
+    if (!isCreateDelegateDto(createDelegateDto)) {
+      throw new HttpException(
+        'Invalid payload',
+        HttpStatus.UNPROCESSABLE_ENTITY,
+      );
+    }
```

Comment:
What about forwarding the error response for the Safe Transaction Service? Doesn't it provide more information on what actually failed (I believe this is the current behaviour of the current CGW).

Reply:
I will check it to return the same in the message :slightly_smiling_face:

Reply:
Ok, I remove then `isCreateDelegate` function to return the error from the `transaction-api` and added a test that check the error is correctly forwarded.

Reply:
Done here https://github.com/5afe/safe-client-gateway-nest/pull/142/commits/90a353171bf0a777e5a2d6828b35e5b80f324a39